addons/web/static/src/js/core/popover.js

```diff
@@ -70,7 +117,14 @@ odoo.define('web.Popover', function () {
                 .find((pos) => {
                     this.popoverRef.el.style.top = `${pos.top}px`;
                     this.popoverRef.el.style.left = `${pos.left}px`;
```

Comment:
Now that I read the code, I understand why intersectionobserver is not what you want. But I don't like the way you do it: you randomly try to position the popover and measure it in the dom (which causes a css redraw), then reposition it until it works.

To be honest, I think that you are solving the issue in the wrong way. This is a common issue, and the solution has always been to do some computations in JS (usually based on fixed width/height/current top/left position ), and then apply the style just once.